etsin_finder/frontend/js/components/qvain/header/index.jsx

```diff
@@ -52,8 +54,13 @@ const QvainHeader = () => {
       </NaviContainer>
       <Right>
         <Settings helpUrl={helpUrl} loginThroughService="qvain" />
+        <DesktopOnly>
+          <LoginButton loginThroughService="etsin" />
```

Comment:
This reference to "etsin" should be "qvain", fixed in different PR here
https://github.com/CSCfi/etsin-finder/commit/e96304ff3aa765b4fb29f22e5b1d72b5a6898a28